Packs/GoogleDocs/Integrations/GoogleDocs/GoogleDocs.py

```diff
@@ -275,6 +275,26 @@ def delete_positioned_object(action_name, object_id):
     }
 
 
+def generate_results(document, human_readable_text):
```

Comment:
It makes discomfort to see that you change what the command functions return and no UT breaks, because thy do not exist.
please consider adding UT to the command functions, if it is a small effort. I will not demand high cover.

Reply:
I've added UTs for the command functions.